static/usage/v7/select/customization/icon-flip-behavior/vue.md

```diff
@@ -0,0 +1,55 @@
+```html
+<template>
+  <ion-list>
+    <ion-item>
+      <ion-select
+        class="always-flip"
+        toggle-icon="caret-down-sharp"
+        interface="popover"
+        label="Icon flips on both modes"
+        placeholder="Select fruit"
+      >
+        <ion-select-option value="apples">Apples</ion-select-option>
+        <ion-select-option value="oranges">Oranges</ion-select-option>
+        <ion-select-option value="bananas">Bananas</ion-select-option>
+      </ion-select>
+    </ion-item>
+    <ion-item>
+      <ion-select
+        class="never-flip"
+        toggle-icon="caret-down-sharp"
```

Comment:
Ionic Vue needs the icons imported from `ionicons/icons` and then passed to the props

```typescript
import { caretDownSharp } from 'ionicons/icons';

...

:toggle-icon="caretDownSharp"
```